bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -209,6 +210,17 @@ class App extends Component {
       />) : null);
   }
 
+  renderUserInformation() {
+    const { UserInfo, User } = this.props;
+
+    return (UserInfo.length > 0 ? (
+      <UserInfoContainer
+        UserInfo={UserInfo}
+        requesterUserId={User.userId}
```

Comment:
Do we need to pull User from props? Isn't this info available in Auth.userID Auth.meetingID ?